concordium-std/Cargo.toml

```diff
@@ -41,6 +41,9 @@ crate-type = ["rlib"]
 [profile.release]
 # Tell `rustc` to optimize for small code size.
 opt-level = "s"
+
+[dev-dependencies]
+trybuild = "1.0.71"
```

Comment:
We usually try not to specify the patch version unless it is critical to do so. Is that necessary here?

Reply:
Dropped patch version. Was just an artifact of `cargo add`.